src/Icon/Icon.jsx

```diff
@@ -24,7 +24,7 @@ export const Icon = ({ name, size, color, rotate, ...props }) => {
     return (
       <BaseIcon
         path={glyphs[name]}
-        size={sizes[size]}
+        size={sizes[size] ?? sizes['small']}
```

Comment:
We should make `size: 'small'` a default prop and here if we don't find the size in the sizes object we should return it
```suggestion
        size={sizes[size] ?? size}
``` 
This will allow us to set custom unit values ( `<Icon size='30px' />` )